docs/api.md

```diff
@@ -341,6 +341,49 @@ const zip = new Blob([data]);
 
 Returns a [`FileSystemTree`](#filesystemtree) when the format is `json`, otherwise a `Uint8Array`.
 
+### ▸ `setPreviewScript`
+
+Added in version `1.5.0`.
+
+Configure a script to be injected inside all previews. After this function returned,
+every preview iframes that are either added or reloaded will now include that extra
+script on all HTML responses.
+
+Notably, existing previews won't included until they have been reloaded.
+
+To reload a preview you can use [`reloadPreview`](#reloadpreview)
+
+:::warning
+This API is a pretty advanced feature that should be used only if this is your only option.
+As you can control what runs in webcontainer it's usually best to use your own server if you can.
```

Comment:
This reads a bit confusingly.

how about something like "Since you can control the content in the pages shown in a preview, it's preferable to add this code when serving the content itself."

Reply:
Oh that's a good idea :+1:

Reply:
sounds good to me!